lib/va1010_forms/enrollment_system/service.rb

```diff
@@ -38,6 +38,14 @@ def submit(parsed_form, form_id)
         }
       rescue => e
         Rails.logger.error "#{form_id} form submission failed: #{e.message}"
+
+        if e.is_a?(Ox::ParseError)
+          PersonalInformationLog.create!(
+            data: submission_body,
+            error_class: 'Form1010Ezr FailedWithFormattingError'
+          )
+        end
```

Comment:
We want to create the `PersonalInformationLog` here because we still have the xml-formatted version of the form.